src/promptflow/promptflow/contracts/tool.py

```diff
@@ -135,6 +135,21 @@ def parse(self, v: Any) -> Any:  # noqa: C901
         # TODO: parse other types
         return v
 
+    def get_class(self):
+        if self == ValueType.INT:
+            return int
+        if self == ValueType.DOUBLE:
+            return float
+        if self == ValueType.BOOL:
+            return bool
+        if self == ValueType.STRING:
+            return str
+        if self == ValueType.LIST:
+            return list
+        if self == ValueType.OBJECT:
+            return dict
+        return None
```

Comment:
Image?

Reply:
We still need to convert for image type.
Will add comment to explain.